blogposts/2021/introducing-sourcegraphs-new-ui.md

```diff
@@ -0,0 +1,64 @@
+
+So with our users and the future in mind, we dove deep into Figma and Excalidraw and sketched concepts in notebooks and maybe on the odd napkin. We presented and reviewed and tested and refined. We searched Sourcegraph thousands of times to discover the scope of impending changes, review code, and answer questions like, "How often is border-color-1 used" or "Who altered this component last?" We created issues, wrote code, read code, deleted code, and merged changesets.
+
+And now, today––a few short months after beginning the project––we are proud to bring you an entirely new visual design for the Sourcegraph UI. Let's take a look at some of the new designs and how they will help users find references, troubleshoot errors, gain insight, make changes on a massive scale, and read code.
+
+## Refined search results design
+
+![refined search results design screenshot](https://sourcegraphstatic.com/blog/redesign/Search%20results.png)
+
+- Searching code can require writing code. The redesigned search bar is 100% width at all times to provide the space required to be expressive with queries.
+- We want to reduce the requirement to learn query syntax. The new results sidebar provides a home for features like dynamic filters that help users discover search syntax without referencing documentation. _It also provides room for new features, so watch this space._
+
+## Renewed focus on code
+
+![focus on code screenshot](https://sourcegraphstatic.com/blog/redesign/code.png)
+
+- Readability of code is paramount. To facilitate that, we’ve reduced non-essential UI elements to provide greater focus on the code itself.
+- Complicated interfaces can cause fatigue, so we've reduced UI elements overall and positioned the most important items so they’re unobtrusive and live exactly where they are needed.
+
+## Improved layouts
+
+![improved diff view screenshot](https://sourcegraphstatic.com/blog/redesign/layouts.png)
+
+- Layout factors heavily in the usability of a design. We've improved pages like diff diff views to make them easier to use and to help users quickly find the information they need. 
+- Segmentation improves focus. We've neatly wrapped sections of the UI in containers to help users focus on the most important elements.
+
+## Consistent headers and navigation
+
+![new campaign headers screenshot](https://sourcegraphstatic.com/blog/redesign/headers.png)
+
+- Sourcegraph is rapidly becoming more than code search. A new global navigation provides immediate discoverability and access to current and future functionality.
+- Consistency improves usability. We've standardized headers to orient users and provide all available options in a single location.
+
+## Promoting extensibility
+
+![exntensions redesign screenshot](https://sourcegraphstatic.com/blog/redesign/exntensions.png)
```

Comment:
@rrhyne this image isn't rendering in the preview for me. Are you having the same issue?

Reply:
I noticed `extensions` was spelled incorrectly in the file path. I just updated it to see if that fixes the issue.